index.php

```diff
@@ -1,14 +1,21 @@
 <?php
-
 /**
  * Plugin Name: Gutenberg
  * Plugin URI: https://wordpress.github.io/gutenberg/
  * Description: Prototyping since 1440. Development plugin for the editor focus in core.
  * Version: 0.1
+ *
+ * @package WordPress
  */
 
+// Hook.
```

Comment:
I would leave it without docs, yes.  I would also move the calls immediately below the functions where they are defined.  This combination of inline docblock, function code, `add_action` is enough to be very clear, IMO.

WP core only requires documentation for the `do_action` or `apply_filters` calls, not the places where the code actually _uses_ a filter ([ref](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters)).